hack/cluster-openshift.sh

```diff
@@ -216,6 +230,20 @@ else
   DOCKER_SUDO=sudo
 fi
 
+# Use curl command if available, otherwise try wget
+if which wget > /dev/null ; then
+  DOWNLOADER="wget -O"
+fi
+if [ ! "$DOWNLOADER" ] ; then
+  if which curl > /dev/null ; then
+    DOWNLOADER="curl -L -o"
+  fi
+fi
+if [ ! "$DOWNLOADER" ] ; then
```

Comment:
Do not abort here if curl or wget is not found - it is possible there is no need to download anything (perhaps the user already downloaded it in a previous run manually?). I would move this down to above line 320, inside the if-stmt when you know you are going to need the DOWNLOADER var.

Reply:
agreed